API/ev3_sensor.c

```diff
@@ -451,7 +451,9 @@ int SetSensorMode(int sensorPort, int name)
 			break;
 		default: return -1;
 	}
-
+	// Set actual device mode
```

Comment:
We have since deprecated `setSensorMode` in favor of `setAllSensorMode`. Does this work for you?